flare/otf.py

```diff
@@ -61,6 +64,11 @@ class OTF:
             "srun".
         dft_kwargs ([type], optional): Additional DFT arguments. Defaults
             to None.
+        store_dft_output (Tuple, optional): After DFT calculations are called,
+            copy the file or files specified in the first element of the tuple
+            to a directory specified as the second element of the tuple.
+            Useful when DFT calculations are expensive and want to be kept
+            for later use.
     """
```

Comment:
Is the first element of the tuple a list of strings? Or just a string? It looks like you've set it up to work with both, but it might be helpful to explicitly state that in the docstring. (You make this very clear in the constructor by giving the type as "Tuple[Union[str,List[str]],str]". Consider doing the same here, so that it shows up on RTD.)